glide-core/src/client/value_conversion.rs

```diff
@@ -66,7 +64,6 @@ pub(crate) fn convert_to_expected_type(
                             _ => Err((
                                 ErrorKind::TypeError,
                                 "Response couldn't be converted to map of {string: double}",
-                                format!("(response was {:?})", map_clone),
```

Comment:
It would be nice if you could still log the type of the response, in order to make it easier to understand falled conversions.